governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+# Maker Executive Aug 2 2023
```

Comment:
The preamble for these is also important, it's displayed on the voting portal in various places.